includes/modules/create_account.php

```diff
@@ -110,6 +110,10 @@
 
   if (ACCOUNT_DOB == 'true') {
     if (ENTRY_DOB_MIN_LENGTH > 0 or !empty($_POST['dob'])) {
+      if (preg_match('/^([0-9]{4})(|-|\/)([0-9]{2})\2([0-9]{2})$/', $dob)) {
+        $_POST['dob'] = date(DATE_FORMAT, strtotime($dob));
+        $dob = date(DATE_FORMAT, strtotime($dob));

```

Comment:
Thanks. 

NOTE: These lines can be converted into one, like this:

```diff
-        $_POST['dob'] = date(DATE_FORMAT, strtotime($dob));
-        $dob = date(DATE_FORMAT, strtotime($dob));
+        $_POST['dob'] = $dob = date(DATE_FORMAT, strtotime($dob));
```

(Note: when doing double assignments like this, the innermost one is done first, so $dob is assigned the result of date() and then $_POST[dob] is assigned what $dob was given. This is seldom "important", but worth knowing.)

Reply:
Tried to make the change so that it would be a squashed commit; however, wasn't successful online.